assets/css/styles.css

```diff
@@ -164,25 +164,37 @@ div.landing-text {
   margin-bottom: 0.67rem;
 }
 
-a.section-button {
+.button {
   font-family: var(--font2);
+  font-size: 0.75em;

```

Comment:
This could be set to the font size variable you have

Reply:
Done